types.go

```diff
@@ -97,15 +97,18 @@ type AuthorizationResponse struct {
 	// the non-raw fields are the best parsings to a single standard, with
 	// loss of granularity minimized. The latter should be preferred when
 	// treating Sleet as a black box.
-	Success              bool
-	TransactionReference string
-	AvsResult            AVSResponse
-	CvvResult            CVVResponse
-	Response             string
-	ErrorCode            string
-	AvsResultRaw         string
-	CvvResultRaw         string
-	RTAUResult           *RTAUResponse
+	Success                  bool
+	TransactionReference     string
+	AvsResult                AVSResponse
+	CvvResult                CVVResponse
+	Response                 string
+	ErrorCode                string
+	AvsResultRaw             string
+	CvvResultRaw             string
+	RTAUResult               *RTAUResponse
+	RecurringDetailReference string
+	ShopperReference         string
```

Comment:
Since ShopperReference is going to be the same as the shopper reference we pass into sleet, do we need to return ShopperReference from sleet?

Reply:
Yeah maybe we should have the sleet.AuthRequest take in a notion of a ShopperReference/ShopperID and then we don't need to necessarily return it back too

Reply:
It is returned back from adyen along with RecurringDetailReference & Alias in the additionalData map. And since now ShopperReference is using UUID maybe just easier to set from adyen response